Assets/_nathan/scripts/GlobalMusic.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using _nathan.scripts;
+using UnityEngine;
+
```

Comment:
Needs Namespace

Reply:
which namespace though? are we combining the scripts into a shared directory later?

Reply:
removed from current pr